dev-infrastructure/docs/development-setup.md

```diff
@@ -122,6 +122,12 @@ This will create:
 1. A custom role definition with required access as defined in `dev-infrastructure/scripts/mock-dev-role-definition.json`
 1. A service principal/application using the created cert as its authentication, and given access based on the custom role definition
 
+In the newly created Resource Group (test-RG):
+1. Create a Virtual Network (test-Vnet) and a Network security group (test-NSG)
```

Comment:
As an improvement these steps can be automated with commands. Not in scope of this PR necessarily